setup/updating.md

```diff
@@ -30,19 +30,27 @@ which you've deployed Coder.
 
 ## Update Coder
 
-Updating Coder is a two-step process:
+To update Coder, follow these steps:
 
 1. Retrieve the latest repository information:
 
     ```bash
     helm repo update
     ```
 
-2. Upgrade to the desired version (i.e., `1.16.0`):
+1. (Optional) Export the current helm chart values into a file:
+
+    ```bash
+    helm get values --namespace coder coder > current-values.yml
+    ```
+
+1. Upgrade to the desired version with your helm chart values file (i.e., `1.16.1`):
+
+    _Note: Omitting `--version` will default to the latest version_
```

Comment:
I thought the first line was a bit confusing, so maybe

> Provide your helm chart values file and upgrade to the desired version (e.g.,
`1.16.1`):
> 
> *Note: If you omit --version, you'll upgrade to the latest version.*

Reply:
I like it, updating!